airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcConnectorMetadataInjector.java

```diff
@@ -14,14 +14,28 @@
 
 public class PostgresCdcConnectorMetadataInjector implements CdcMetadataInjector<Long> {
 
+  private final String transactionTimestamp;
```

Comment:
is this related to meta support?

Reply:
No, it's a refactor to get rid of an extra class